pkg/aws/policies.go

```diff
@@ -738,9 +742,17 @@ func (c *awsClient) DeleteOperatorRole(roleName string) error {
 					roleName)
 			}
 		}
-		return err
+		return "", err
 	}
-	return c.DeleteRole(roleName, role)
+	policyARN, err := c.deleteOperatorRolePolicies(policies)
```

Comment:
Do we need a similar function for deleting account role policies?
@pvasant @vkareh just double checking - we want to delete the account role policies after the account roles are removed, right?

Reply:
This is a good point. The function that deletes the roles, detaches the policies but doesn't delete them. It only attempts deleting inline policies, but we're not using inline policies for this. The policies should be deleted after being detached.